File: uri/1 - Basics/1071/Main.java
import java.util.*;
class Main{
    public static void main(String [] args){
        Scanner scanner = new Scanner(System.in);
        int x = scanner.nextInt();
        int y = scanner.nextInt();
        int aux;
        if(x > y){
         aux = x;
         x = y;
         y = aux;
        }
        int soma = 0;
        for(int i = x+1; i < y; i++){
            if(i%2 != 0){
               soma += i;
            }
        }
        System.out.printf("%d\n", soma);
    }
}

File: uri/1 - Basics/1071/Main.go
package main

import "fmt"

func main() {
	var x, y, i, aux int
	soma := 0

	fmt.Scanf("%d", &x)
	fmt.Scanf("%d", &y)

	if x > y {
		aux = x
		x = y
		y = aux
	}

	for i = x + 1; i < y; i++ {
		if i%2 != 0 {
			soma += i
		}
	}

	fmt.Printf("%d\n", soma)
}
